shamcoffee/shamcoffee.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>shamcoffeehouse</title>
+    <link rel="stylesheet" href="shamcoffee.css">
+</head>
+<body>
+    <div class="header">
```

Comment:
Let's add some z-index to the header so that the animated coffee cup doesn't hide it